### PR TITLE
periph/rtt: add macros for time to tick conversion

### PR DESCRIPTION
**Pull request by daniel-k**
I added macros for conversion up to minutes, not sure if anything above would be useful.

Additionally, I don't have any macros for conversion from ticks to time. Do we need them?

### CODE REVIEW

File: drivers/include/periph/rtt.h [hunk 1]
```diff
@@ -32,6 +32,39 @@ extern "C" {
...
+#warning "RTT_FREQUENCY undefined. Set RTT_FREQUENCY to the number of ticks" \
+         "per second for the current architecture."
+#endif
+
+/**
+ * @brief       Convert microseconds to rtt ticks
+ * @param[in]   us number of microseconds
+ * @return      rtt ticks
+ */
+#define RTT_US_TO_TICKS(us)     (uint32_t)((uint64_t)(us) * RTT_FREQUENCY / 1000000UL )
```
Comment (Kijewski) [inline drivers/include/periph/rtt.h]: Please surround with parens.

**Comment by daniel-k** [comment]
Surrounding done.

File: drivers/include/periph/rtt.h [hunk 2]
```diff
@@ -32,6 +32,39 @@ extern "C" {
...
+ * @return      rtt ticks
+ */
+#define RTT_US_TO_TICKS(us)     ((uint32_t)((uint64_t)(us) * RTT_FREQUENCY / 1000000UL))
+
+/**
+ * @brief       Convert milliseconds to rtt ticks
+ * @param[in]   ms number of milliseconds
+ * @return      rtt ticks
+ */
+#define RTT_MS_TO_TICKS(ms)     ( RTT_US_TO_TICKS((ms) * 1000) )
```
Comment (Kijewski) [inline drivers/include/periph/rtt.h]: The calculation would be a little more stable if you did not yield to `RTT_US_TO_TICKS` but use `((uint32_t)((uint64_t)(us) * RTT_FREQUENCY / 1000UL))` directly. Same for the macros below.

Comment (daniel-k) [inline drivers/include/periph/rtt.h]: Could you please provide an example? Calculation is done from left to right and the bigger the value before division, the more stable it is, isn't it? Please note that ` * 1000` is done inside macro argument

Comment (Kijewski) [inline drivers/include/periph/rtt.h]: Sorry, I managed to to misunderstand the calculation. -_- Please ignore my comment.

**Comment by daniel-k** [comment]
@Kijewski 
Ping. Any objections? Shoud I also add macros for conversion from ticks to time?

**Comment by daniel-k** [comment]
@thomaseichinger ping

**Comment by thomaseichinger** [comment]
Looks good to me, triggered CI builds.

**Comment by daniel-k** [comment]
@thomaseichinger 
It seems like Travis needs to be restarted. I don't know how to do so :/

**Comment by OlegHahm** [comment]
@daniel-k, klick on "Details", login with your Github account and click the restart button on the upper right corner.

**Comment by OlegHahm** [comment]
travis_restart

**Comment by daniel-k** [comment]
@thomaseichinger Was this an ACK? We could merge then.

**Comment by thomaseichinger** [comment]
ACK & Go

**Pull request merged**